oneoffs/joseki/opening_freqs.py

```diff
@@ -122,8 +186,19 @@ def extract_corners(game_path):
         _, moves = sgf_moves.get_setup_and_moves(g)
     except BaseException:
         print("bad file: ", game_path)
-        return Counter()
+        return Counter(), {}
+
+    return (extract_corners(moves), 1 if g.get_winner() == 'b' else 0)
```

Comment:
what is get_winner() from?
should we .lower() just to be safe?

Reply:
it's from sgfmill, it seems to call lower on it already.

Reply:
Done.